### PR TITLE
✨ Initial Biospecimen Conformance Resources

### LINKED ISSUE #25: Develop & test Kids First biospecimen resources
- [ ] Create StructureDefinition if needed
- [ ] Create any necessary Extensions for added attributes
- [ ] Create SearchParameters for the Extensions from ^
- [ ] Create any necessary CodeSystems or ValueSets

### PR DESCRIPTION
**Pull request by liberaliscomputing**
Closes #25 

**Dependencies:**

- [x] StructureDefinition `StructureDefinition/biospecimen`
- [x] Extension `StructureDefinition/concentration`
- [x] `SearchParameter/concentration`

**To do:**

- [x] Add the above extension to Biospecimen StructureDefinition
- [x] Add `Specimen-bs-001.json` with the extended attribute populated

**Mapping Spreadsheet:**

https://docs.google.com/spreadsheets/d/19tQnE75UzvP_k29D-QprbsJ-6ZO2PdUmKPiWHKkcTEg/edit#gid=1197884015

### CODE REVIEW

**Comment by liberaliscomputing** [comment]
@znatty22, re-structured!

File: site_root/input/resources/examples/Specimen-bs-001.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (znatty22) [inline site_root/input/resources/examples/Specimen-bs-001.json:1]: Can you make this example as complete as possible? What I mean is, let's try to have all of our examples fill out as many of the resource attributes as possible so people can see how to use them. For example, I see this is missing `collection.method`, `collection.bodySite`, `receiveTime` etc.

Comment (liberaliscomputing) [reply site_root/input/resources/examples/Specimen-bs-001.json:1]: No problem. Per `age_at_event_days`, we can import your `age-at-event` extension once the phenotype PR is merged in.

Per `analyte_type`, we discussed that this should be mapped into a new `sequencing_experiment` profile using FHIR MolecularSequence, so the unnecessary attributes to the essence of a specimen such as  `analyte_type` and `sequencing_center_id` can be separated. That's why this wasn't included.

Per `consent_type`, we also wanted to discuss whether it should be at a specimen-level or we'd want to use FHIR Consent, instead.

I will create two tickets about these.

File: site_root/input/resources/examples/Specimen-bs-001.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
```
Comment (znatty22) [inline site_root/input/resources/examples/Specimen-bs-001.json:1]: Can you fill out the `bodySite` in the example, just to make a little bit more complete.

Comment (liberaliscomputing) [reply site_root/input/resources/examples/Specimen-bs-001.json:1]: Updated and validated!

**Pull request merged**